functions/isbetween.h

```diff
@@ -0,0 +1,60 @@
+#ifndef FUNCTIONS_ISBETWEEN_H
+#define FUNCTIONS_ISBETWEEN_H
+
+// Usage: IsBetween<F, BOTTOM, TOP>
+// Returns 0 or 32768 based F > BOTTOM and < TOP 
+// F, BOTTOM, TOP: INTEGER
+// return value: INTEGER
+
+class BladeBase;
+
+template<class F, class BOTTOM, class TOP>
+class IsBetweenBase {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    bottom_.run(blade);
+    top_.run(blade);
+  }
+  int getInteger(int led) {
+    if (f_.getInteger(led) > bottom_.getInteger(led) && f_.getInteger(led) < top_.getInteger(led)) {
```

Comment:
We need to use a temporary variable instead of calling f_.getInteger(led) twice.

Reply:
added